collector/src/descriptor/kind/bridge_server_descriptor.rs

```diff
@@ -49,6 +73,40 @@ impl BridgeServerDescriptor {
             )
         };
 
+        let master_key = {
+            let v = take_uniq(&mut desc, "master-key-ed25519", 1)?;
+            v[0].to_owned()
+        };
+
+        let (ipv6, or_port_v6) = {
+            let v = take_opt(&mut desc, "or-address", 1)?;
+            if let Some(t) = v.map(|x| x[0].split("]:").collect::<Vec<_>>()) {
```

Comment:
Could you use [`SocketAddrV6::parse`](https://doc.rust-lang.org/std/net/struct.SocketAddrV6.html) instead of manual splitting?